includes/admin/functions.php

```diff
@@ -69,3 +52,78 @@ function amp_admin_get_preview_permalink() {
 
 	return amp_get_permalink( $post_id );
 }
+
+/**
+ * Registers a submenu page to access the AMP template editor panel in the Customizer.
+ */
+function amp_add_customizer_link() {
+	// Teensy little hack on menu_slug, but it works. No redirect!
+	$menu_slug = add_query_arg( array(
+		'autofocus[panel]'         => AMP_Template_Customizer::PANEL_ID,
+		'return'                   => rawurlencode( admin_url() ),
+		AMP_CUSTOMIZER_QUERY_VAR   => true,
+	), 'customize.php' );
+
+	// Add the theme page.
+	add_theme_page(
+		__( 'AMP', 'amp' ),
+		__( 'AMP', 'amp' ),
+		'edit_theme_options',
+		$menu_slug
+	);
+}
+
+/**
+ * Registers a top-level menu for AMP configuration options
+ */
+function amp_add_options_menu() {
+	$amp_options = new AMP_Options_Menu();
+	$amp_options->init();
+}
+add_action('admin_menu','amp_add_options_menu');
```

Comment:
For any new code, let's be mindful of WordPress coding standards around spacing and indents: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

e.g. this would become 

```
add_action( 'admin_menu', 'amp_add_options_menu' );
```

Would be good to especially clean this up in any new files.